src/pags/auth/login.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Link, useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [message, setMessage] = useState('');
+  const navigate = useNavigate()
+  const handleSubmit = async (e) => {
+  e.preventDefault();
+  try {
+      const response = await axios.post('http://localhost:8000/login', {
+        email: email,
+        password: password
+      });
+      setMessage(response.data.message);
+
+      if(response.status == 200){
+        localStorage.setItem('user',response.data.user)
+        window.location = "/chat"
+      }
+  } catch (error) {
+    
+      setMessage(error.response.data.detail);
+  }
+  
+  };
+
+  return (
+    <div className="container mt-5">
+  <div className="row justify-content-center">
+    <div className="col-sm-8 col-md-6 col-lg-5">
+      <div className="border p-3" style={{ borderRadius: 4 }}>

```

Comment:
never inline style , sorry. thay might ask u why read about classnames wights and scoring system